g/gson/gson_ubi_9.3.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+#
+# Package       :gson
+# Version       : 2.10.1
+# Source repo   : https://github.com/google/gson.git
+# Tested on     : UBI 9.3
+# Language      : Java
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Pratibh Goshi<pratibh.goshi@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+set -e
+
+PACKAGE_NAME=gson
+PACKAGE_VERSION=${1:-gson-parent-2.10.1}
+PACKAGE_URL=https://github.com/google/gson.git
+
+# install tools and dependent packages
+yum install -y git wget
+
+# setup java environment
+yum install -y java java-devel
+
+export JAVA_HOME=/usr/lib/jvm/$(ls /usr/lib/jvm/ | grep -P '^(?=.*java-)(?=.*ppc64le)') 
+# update the path env. variable
+export PATH=$PATH:$JAVA_HOME/bin
+
+# install maven
+MAVEN_VERSION=${MAVEN_VERSION:-3.8.8}
+wget https://downloads.apache.org/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz
+tar -C /usr/local/ -xzf apache-maven-$MAVEN_VERSION-bin.tar.gz
+mv /usr/local/apache-maven-$MAVEN_VERSION /usr/local/maven
+
+export M2_HOME=/usr/local/maven
+
+# update the path env. variable
+export PATH=$PATH:$M2_HOME/bin
+
+# clone and checkout specified version
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
```

Comment:
Need to cd in the gson directory as the outer one is gson-parent.
https://github.com/google/gson/blob/main/gson/pom.xml